### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -113,6 +113,52 @@ SOFTWARE.
...
+## @react-native-community/masked-view
+
+This product contains '@react-native-community/masked-view' by React Native Community.
+
+React Native Masked View Library
+
+* HOMEPAGE:
+  * https://github.com/react-native-community/react-native-masked-view
+
+* LICENSE:
```
Comment (amyblais) [inline NOTICE.txt:158]: No license

File: NOTICE.txt [hunk 2]
```diff
@@ -1605,6 +1716,19 @@ SOFTWARE.
...
+## react-native-file-viewer
+
+This product contains 'react-native-file-viewer' by Vincenzo Scamporlino.
+
+Native file viewer for React Native. Preview any type of file supported by the mobile device.
+
+* HOMEPAGE:
+  * https://github.com/vinzscam/react-native-file-viewer
+
+* LICENSE:
```
Comment (amyblais) [inline NOTICE.txt:1728]: No license in their repo

**Comment by amyblais** [comment]
Elias, I'm re-requesting your review because there are two repos that don't have a licence. Will need to determine next steps here.

File: NOTICE.txt (hunk 1 above)
Comment (enahum) [reply NOTICE.txt:158]: this one is from `react-native-community` so is fine to use, no license file but it is MIT 
image

File: NOTICE.txt (hunk 2 above)
Comment (enahum) [reply NOTICE.txt:1745]: I'd say lets add an issue in their repo and ask for the license, if it ends up being not compatible then we can remove it and look for an alternative. for now we can keep.

Comment (amyblais) [reply NOTICE.txt:1745]: @grundleborg Please help confirm if this is the best next step here.

Comment (grundleborg) [reply NOTICE.txt:1745]: I can't really comment. Perhaps ask @wiersgallak for advice?

Comment (amyblais) [reply NOTICE.txt:1745]: License request: https://github.com/vinzscam/react-native-file-viewer/issues/79.

Comment (amyblais) [reply NOTICE.txt]: License was added.

**Pull request merged**